rssalertbot

```diff
@@ -348,13 +380,13 @@ def guess_color(matchstring):
 
     for key in KEYS_GREEN:
         if key in matchstring:
-            return 'green'
+            return {"hipchat_color": "green", "slack_color": "#3aa459"}
```

Comment:
I'd be tempted to just use the slack default color names here, the `danger` and `warning` names.

Reply:
that's good suggestion - added good / warning / danger